### ISSUE TITLE
Add step to remove gparted from vbox before starting up the boot2docker image

### ISSUE THREAD

**Issue by mryuha** [docker/docker#9237]
https://github.com/docker/docker/blob/master/docs/sources/articles/b2d_volume_resize.md

**Comment by jfrazelle** [comment]
@SvenDowideit is this needed in docs

**Comment by MHBauer** [comment]
Is this issue still relevant? Looking back in the history I can see at the time the documentation did mention removing the iso from the IDE controller before rebooting.

Perhaps this refers to screenshots for the process? Nearly all the steps for interacting with virtualbox have screenshots.

**Comment by mryuha** [comment]
No longer needed
